src/game/level_update.c

```diff
@@ -1033,9 +1033,16 @@ s32 play_mode_paused(void) {
         if (gDebugLevelSelect) {
             fade_into_special_warp(WARP_SPECIAL_LEVEL_SELECT, 1);
         } else {
+        #ifdef DEATH_ON_EXIT_COURSE
```

Comment:
I wouldn't indent preprocessor checks, for the sake of matching the format of the rest of the repository. The main exception to this IMO is in safeguards because they're littered everywhere in that file.